.cirrus.yml

```
@@ -34,36 +44,64 @@ test_task:
     - wget -O otp.tar.gz https://repo.hex.pm/builds/otp/ubuntu-14.04/${OTP_RELEASE}.tar.gz
     - mkdir -p otp
     - tar zxf otp.tar.gz -C otp --strip-components=1
-    - otp/Install -minimal $(pwd)/otp
-
-  test_script:
-    - PATH=$(pwd)/otp/bin:$PATH
+    - otp/Install -minimal ${CIRRUS_WORKING_DIR}/otp
     - rm -rf .git
-    - ELIXIRC_OPTS="--warnings-as-errors" ERLC_OPTS="+warning_as_errors" make compile
-    - make test
-    - dialyzer -pa lib/elixir/ebin --build_plt --output_plt elixir.plt --apps lib/elixir/ebin/elixir.beam lib/elixir/ebin/Elixir.Kernel.beam
-
-    # Check for reproducible builds only in the latest OTP release
-    - if [ -n "$CHECK_REPRODUCIBLE" ]; then make check_reproducible; fi
-
-    # Check for POSIX compliant shell scripts
-    - if [ -n "$CHECK_POSIX_COMPLIANT" ]; then
-        apt update;
-        apt install -y shellcheck;
-        shellcheck -e SC2039,2086 bin/elixir && echo "bin/elixir is POSIX compliant";
-        shellcheck bin/elixirc && echo "bin/elixirc is POSIX compliant";
-        shellcheck bin/iex && echo "bin/iex is POSIX compliant";
-      fi
+    - make compile
+
+  info_script:
+    - bin/elixir --version
+
+  test_formatted_script: |
+    if [ -n "$TEST_FORMATTED" ]; then
+      make test_formatted && echo "All files found to be properly formatted."
+    else
+      echo "No need for this test in this Erlang/OTP version."
```

Nitpick: instead of saying "No need for this test", we can say where to find this test:

```suggestion
      echo "The format of the code is checked in the last stable Erlang/OTP version."
```